packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md

```diff
@@ -10,46 +10,39 @@ This guide covers how to upgrade the Okta Sign-In Widget, which depends on wheth
 
 **Learning outcomes**
 
-* Understand the benefits of keeping your widget up to date.
-* Know how to obtain the latest version of the widget.
 * Update your widget based on your user deployment model.
 
 **What you need**
 
-[Latest available widget release](https://github.com/okta/okta-signin-widget/releases)
+* An [Identity Engine-upgraded Okta org](/docs/guides/oie-upgrade-overview/)
+* An existing app that uses the Okta Sign-In Widget
+* [Latest available widget release](https://github.com/okta/okta-signin-widget/releases)
 
 ---
 
 ## About the widget
 
 The widget is a JavaScript library that gives you a full-featured and customizable sign-in experience that you can use to authenticate users of web and mobile applications.
 
-The widget is used on Okta's default sign-in page to start an Okta SSO session and to set the Okta [session cookie](/docs/guides/session-cookie/) in the web browser. The widget can also perform a complete [OpenID Connect (OIDC)](/docs/reference/api/oidc/) flow and/or integrate with [external Identity Providers](/docs/concepts/identity-providers/).
+The widget is used on Okta's default sign-in page to start an Okta SSO session and to set the Okta [session cookie](/docs/guides/session-cookie/) in the web browser. The widget can also perform a complete [OpenID Connect (OIDC)](/docs/concepts/oauth-openid/) flow and/or integrate with [external Identity Providers](/docs/concepts/identity-providers/).
 
 This article teaches you how to upgrade the widget when it is used in any of the following ways:
 
 * Okta-hosted sign-in page (default): Okta provides a sign-in page that is available at your [org's URL](/docs/concepts/okta-organizations/). By default, a user who signs in on this page is redirected to the Okta user dashboard.
```

Comment:
Are we supposed to stop using "hosted"? If so, second bullet should have two references removed, and first bullet one reference. If not, I'd add "hosted" to the 1st bullet as part of the sentence for symmetry with the 2nd bullet.